feed.rb

```diff
@@ -1,16 +1,18 @@
 require 'bundler'
 Bundler.require
 
-require './conda_repo'
+require './conda'
 
-# redis_uri = ENV['REDIS_SERVER'] || 'localhost:6379'
-# client = Feedtosis::Client.new('https://repo.anaconda.com/pkgs/rss.xml',
-#                                backend: Moneta.new(:Redis,
-#                                                    host: URI.parse(redis_uri).host,
-#                                                    port: URI.parse(redis_uri).port))
-# TODO: do this in next PR
-# while(true) do
-#  new_entries = client.fetch.new_entries
-#  CocoapodsRepo.new(redis_uri).update_pods if new_entries
-#  sleep 30
-# end
+redis_uri = ENV['REDIS_SERVER'] || 'localhost:6379'
+client = Feedtosis::Client.new('http://repo.anaconda.com/pkgs/rss.xml',
```

Comment:
Is this available over https?

Reply:
it *IS* but then feedtosis breaks :(  https://github.com/jsl/feedtosis/blob/master/lib/feedtosis/client.rb#L39